auth0/src/test/java/com/auth0/android/authentication/AuthenticationAPIClientTest.java

```diff
@@ -120,12 +120,12 @@ public void shouldSetUserAgent() {
 
     @Test
     public void shouldSetTelemetryIfPresent() {
-        final Telemetry telemetry = mock(Telemetry.class);
-        when(telemetry.getValue()).thenReturn("the-telemetry-data");
+        final Auth0UserAgent auth0UserAgent = mock(Auth0UserAgent.class);
+        when(auth0UserAgent.getValue()).thenReturn("the-telemetry-data");
```

Comment:
I'd also take the chance to update this mocked string value and also the tests' names

Reply:
👍 New commit to address both.